test/generate-testcase.js

```diff
@@ -54,7 +55,10 @@ function fetchSource(url, callbackFn) {
   if (url.indexOf("https") == 0) {
     client = require("https");
   }
-  client.get(url, function(response) {
+  var options = urlparse(url);
+  options.headers = {'User-Agent': 'readability.js'}
```

Comment:
I'm not actually sure we should be doing it this way, or should create a new object instead of adding properties to the one returned by urlparse.